auth/tenant_mgt.go

```diff
@@ -240,6 +242,11 @@ func (t *TenantToCreate) EnableEmailLinkSignIn(enable bool) *TenantToCreate {
 	return t.set(enableEmailLinkSignInKey, enable)
 }
 
+// EnableAnonymousUsers enables or disables anonymous user.
```

Comment:
Looks like `EnableAnonymousUsers` is a literal that should be backticked for code font.

Reply:
It doesn't look like we back-tick literals in Go SDK. ex: `TenantClient`, `AndroidNotification` etc.